### PR TITLE
Create new rules

### PR DESCRIPTION
**Pull request by jorisdebock**
Linustechtips.com.xml
Linusmediagroup.com.xml
Powned.tv.xml
Ted.com.xml

### CODE REVIEW

File: src/chrome/content/rules/Linusmediagroup.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Linusmediagroup">
+  <target host="www.linusmediagroup.com" />
```
Comment (2d1) [inline src/chrome/content/rules/Linusmediagroup.com.xml]: Certificate doesn't match.

File: src/chrome/content/rules/Linusmediagroup.com.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+  <test url="http://www.linusmediagroup.com/" />
```
Comment (2d1) [inline src/chrome/content/rules/Linusmediagroup.com.xml]: This test is redundant, as literal targets generate implicit tests.

File: src/chrome/content/rules/Linustechtips.com.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Linustechtips">
```
Comment (2d1) [inline src/chrome/content/rules/Linustechtips.com.xml]: This should target www.linustechtips.com, which works with a direct rewrite.

File: src/chrome/content/rules/Linustechtips.com.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  <target host="linustechtips.com" />
+
+  <test url="http://linustechtips.com/" />
```
Comment (2d1) [inline src/chrome/content/rules/Linustechtips.com.xml]: This test is redundant, as literal targets generate implicit tests.

File: src/chrome/content/rules/Ted.com.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Ted">
```
Comment (2d1) [inline src/chrome/content/rules/Ted.com.xml]: Please make additions and changes for ted.com to TED.com.xml, and more generally grep all rulesets for existing domains before making a new one.

File: src/chrome/content/rules/Ted.com.xml [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+  <target host="www.ted.com" />
+
+  <test url="http://www.ted.com/" />
```
Comment (2d1) [inline src/chrome/content/rules/Ted.com.xml]: This test is redundant, as literal targets generate implicit tests.

File: src/chrome/content/rules/powned.tv.xml [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Powned">
```
Comment (2d1) [inline src/chrome/content/rules/powned.tv.xml]: This ruleset should target ^powned.tv.

File: src/chrome/content/rules/powned.tv.xml [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+  <target host="www.powned.tv" />
+
+  <test url="http://www.powned.tv/" />
```
Comment (2d1) [inline src/chrome/content/rules/powned.tv.xml]: This test is redundant, as literal targets generate implicit tests.

**Comment by semenko** [comment]
Looks like this PR has some outstanding edits -- and it hasn't been updated in a while. I'm going to close this for now, but if you'd like it merged, feel free to re-open. Thanks! :)

**Pull request closed without merging**